__tests__/app.test.js

```diff
@@ -130,6 +130,7 @@ describe("/api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "11",
         });
       });
```

Comment:
I think there should be a check that if the comment_count is 0 then it behaves normally and doesn't give anything unexpected